doc/ensemble.rst

```diff
@@ -0,0 +1,739 @@
+********
+Ensemble
+********
+
+========
+Overview
+========
+An ``Ensemble`` is a SmartSim object comprised of multiple
+:ref:`Model<model_api>` objects, each representing an individual
+simulation. An Ensemble can be be reference as a single instance and
+enables users to launch a group of simulations
+at runtime. Ensembles can be launched with other SmartSim
+entities and ML infrastructure to build AI-enabled workflows.
+The :ref:`Ensemble API<ensemble_api>` offers key Ensemble features,
+including helper functions to:
```

Comment:
cue my "helper functions" hate. it's the Ensemble API. a helper is `def format_an_ensemble_for_tv()`